src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -198,7 +196,7 @@ public void It_publishes_projects_with_filter_and_rid()
                 $"{FileConstants.DynamicLibPrefix}coreclr{FileConstants.DynamicLibSuffix}"
             });
 
-            publishDirectory.Should().NotHaveFiles(new[] {
+            publishDirectory.Should().HaveFiles(new[] {
```

Comment:
Should the version in the filter profile be updated instead? https://github.com/dotnet/sdk/tree/main/src/Assets/TestProjects/StoreManifests

@dsplaisted , how do filterprofiles work? I assume that if the profile was updated to 13.0.1 then the publish directory would no longer include newtonsoft (which is what this test is testing).